ppoprf/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ppoprf"
-version = "0.0.1"
+version = "0.1.0"
```

Comment:
where can i find a record of that?

Reply:
Brave [vendors](https://github.com/brave/brave-core/blob/v1.52.13/DEPS#L28) web-discovery-project, with [refers to](https://github.com/brave/web-discovery-project/blob/9469f5e76d5936f1f5f1dd06b9e312a1f5677ec4/package.json#L124) version 0.1.3 of star-wasm. So I flipped the minor and tiny revs. So 0.2.0 would be ok if you hate the gap, but we've been making test deployments with various versions of the ppoprf crate since then, so I still think 0.5.x is a reasonable place to bump to.

Since the crates in the repo all depend on each other I'd like to keep their versions aligned, even though wdp doesn't actually use the ppoprf crate.

Reply:
so 0.1.3 refers to the star-wasm crate, and not the ppoprf crate? in other words, there was never a 0.1.0 of the ppoprf crate? I'm unsure if it makes sense to skip 1 to 4 minor versions because of various test deployments. I would imagine it would be confusing for a future dev, if they wanted to go back and find 0.1.x for whatever reason. Choosing 0.1.x would also keep all crate versions aligned perfectly

Reply:
Version numbers are arbitrary, it doesn't really matter, but I think it's more confusing if the wdp 0.1.3 release is a year and half older than the 0.1.0 release from the same repo.

Please just bump it to 0.2.0 or something else past 0.1.x.

Reply:
> I think it's more confusing if the wdp 0.1.3 release is a year and half older than the 0.1.0 release from the same repo

the two crates are independent of each other. consider this example scenario: if we set ppoprf to 0.2.0 and star-wasm went through multiple iterations and eventually landed at 0.5.0, would we be expected to bump a new iteration of ppoprf to 0.6.0? 

it would be best to set a consistent guideline for bumping semvers in this repo. imo, it would be less confusing (and more semantic) to bump each crate version independently. if we eventually published these crates to crates.io for outsiders to use, they may find it confusing and unorthodox that the major or minor versions include random intervals in between.

Reply:
Either that, or we bump every crate to 0.2.0, and make a standard to bump every crate version, no matter where the change is.

Reply:
That's fine with me. The corresponding git tags are repo-wide.